### PR TITLE
Final IESG nits

### PR DESCRIPTION
**Pull request by martinduke**
Defined MASQUE, shortened Multicast sentence, fixed line endings.

### CODE REVIEW

File: proposed_charter.md [hunk 1]
```diff
@@ -18,24 +18,24 @@ different services.
...
 The primary goal of this working group is to develop mechanism(s) that allow
 configuring and concurrently running multiple proxied stream- and datagram-based
-flows inside an HTTPS connection. The group will specify HTTP and/or HTTP/3
-extensions to enable this functionality. The group will focus on a limited set
-of client-initiated services: (1) UDP CONNECT and (2) IP proxying. 
-Server-initiated services are out of scope. The working group will first deliver
-a protocol solution for UDP CONNECT and a requirements document for IP proxying.
-Once both are complete, the working group will focus on a protocol solution for 
-IP proxying.
+flows inside an HTTPS connection. This framework is called MASQUE. The group
```
Comment (chris-wood) [inline proposed_charter.md]: What is "this framework"? 

Maybe we should rewrite the intro to something like the following:

```
The primary goal of this working group is to develop a framework for configuring and concurrently 
running multiple proxied stream- and datagram-based flows inside an HTTPS connection. 
This framework is called MASQUE. The group....
```

**Review by chris-wood — APPROVED** [review]
LGTM modulo one comment.

**Review by erickinnear — APPROVED** [review]

**Review by ekline — APPROVED** [review]
LGTM.

My only comment was that MASQUE appeared in the text without prior definition.

**Review by chris-wood — APPROVED** [review]

**Review by DavidSchinazi — COMMENTED** [review]
LGTM

**Pull request merged**